app/src/main/java/ch/hikemate/app/model/authentication/FirebaseAuthRepository.kt

```diff
@@ -203,35 +178,108 @@ class FirebaseAuthRepository : AuthRepository {
   private fun reauthenticate(
```

Comment:
Nitpick: Could you please include a docstring? I know its not your function, but I saw that its missing.

Reply:
Sure!